src/emitter.js

```diff
@@ -22,7 +22,7 @@ const strongDirRegExp = new RegExp(
 
 class BananaEmitter {
   constructor (locale) {
-    this.locale = locale
+    this.locale = normalizeLocale(locale)
```

Comment:
Here the normalized version of `locale` is assigned to `this.locale` but on the following line, the not-normalized version of `locale` is used.

Reply:
Fixed.